test/mango.py

```diff
@@ -96,6 +96,8 @@ def create_index(self, fields, idx_type="json", name=None, ddoc=None):
         body = json.dumps(body)
         r = self.sess.post(self.path("_index"), data=body)
         r.raise_for_status()
+        assert r.json()["id"] != None
+        assert r.json()["name"] != None
```

Comment:
`foo is None` is how values should be checked in Python against singletons which are None, False, True and Ellipsis.